From a201686e3f03c6eade379f4ac9b4ce8b4a79aa5e Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Wed, 12 Sep 2012 07:03:27 -0400
Subject: [PATCH] Use auth_admin, not auth_self

To execute a program as root, the user needs to authenticate himself as
an administrator. Allowing any non-administrative user to run any
program as root just by entering his own password is completely
unacceptable on shared machines.
---
 data/org.gnome.gksu.policy | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/org.gnome.gksu.policy b/data/org.gnome.gksu.policy
index ff0e418..ec8e463 100644
--- a/data/org.gnome.gksu.policy
+++ b/data/org.gnome.gksu.policy
@@ -10,9 +10,9 @@
     <description>spawn</description>
     <message>System policy prevents executing a program with administration privileges</message>
     <defaults>
-      <allow_any>auth_self</allow_any>
-      <allow_inactive>auth_self</allow_inactive>
-      <allow_active>auth_self</allow_active>
+      <allow_any>auth_admin</allow_any>
+      <allow_inactive>auth_admin</allow_inactive>
+      <allow_active>auth_admin</allow_active>
     </defaults>
   </action>
 
-- 
1.7.12

